controller/controller_server/controller_types.py

```diff
@@ -0,0 +1,23 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class ArrayConnectionInfo:
+    user: str
+    password: str
+    array_addresses: list

```

Comment:
I think `array_addresses` should be first, as in the original secret of our yaml examples - it's more aligned with their logical hierarchy

Reply:
done